packages/grafana-e2e/cypress.json

```diff
@@ -1,5 +1,7 @@
 {
   "projectId": "zb7k1c",
   "supportFile": "cypress/support/index.ts",
-  "videoCompression": 20
+  "videoCompression": 20,
+  "viewportWidth": 1920,
```

Comment:
doesn't this just get overwritten by https://github.com/grafana/grafana/blob/main/packages/grafana-e2e/cypress/plugins/index.js#L43 anyway?